sass/components/list.sass

```diff
@@ -0,0 +1,22 @@
+$list-border: 1px solid $border !default
+$list-background-color: $white !default
+$list-radius: $radius !default
+
+$list-item-border: 1px solid $border !default
+$list-item-hover-color: $background !default
+
+.list
+  background-color: $list-background-color
+  border: $list-border
+  border-radius: $list-radius
+  .list-item
+    color: black
```

Comment:
Use `color: $list-item-color` and set the default value to `$text`.

Reply:
oh yes, my fault :O